src/main/java/fi/otavanopisto/kuntaapi/server/integrations/vcard/VCardTranslator.java

```diff
@@ -0,0 +1,271 @@
+    if (types != null && !types.isEmpty()) {
+      return types.get(0).getValue();
+    }
+
+    return null;
+  }
+
+  private String getPhoneType(List<TelephoneType> types) {
+    if (types != null && !types.isEmpty()) {
+      return types.get(0).getValue();
+    }
+
+    return null;
+  }
+
+  private List<LocalizedValue> getLocalizedString(String value) {
+    if (StringUtils.isNotBlank(value)) {
+      List<LocalizedValue> result = new ArrayList<>(1);
+      
+      LocalizedValue localizedValue = new LocalizedValue();
+      localizedValue.setLanguage(VCardConsts.DEFAULT_LANGUAGE);
+      localizedValue.setValue(value);
+      
+      result.add(localizedValue);
+      
+      return result;
+    }
+
+    return Collections.emptyList();
+  }
+
+  private String getOrganization(Organization organization) {
+    if (organization != null && organization.getValues() != null && !organization.getValues().isEmpty()) {
+      return organization.getValues().get(0);
+    }
+    
+    return null;
+  }
+  
+  private List<String> getOrganizationUnits(List<Organization> organizations) {
+    if (organizations != null && organizations.size() > 1) {
+      List<String> result = new ArrayList<>(organizations.size() - 1);
+      for (int i = 1, l = organizations.size(); i < l; i++) {
+        Organization organization = organizations.get(i);
+        String value = organization.getValues().isEmpty() ? null : organization.getValues().get(0);
+        if (StringUtils.isNotBlank(value)) {
+          result.add(value);
+        }
+      }
+      
+      return result;
+    }
+    
+    return Collections.emptyList();
+  }
+
+  private String getFirstName(StructuredName structuredName) {
+    if (structuredName != null && StringUtils.isNotBlank(structuredName.getGiven())) {
+      return structuredName.getGiven();
+    }
+    
+    return null;
+  }
+  
+  private String getLastName(StructuredName structuredName) {
+    if (structuredName != null && StringUtils.isNotBlank(structuredName.getFamily())) {
+      return structuredName.getFamily();
+    }
+    
+    return null;
+  }
+
+  private String getString(String value) {
+    return getString(value, null);
+  }
+  
+  private String getString(String value, String defaultValue) {
+    if (StringUtils.isNotBlank(value)) {
+      return value;
+    }
+    
+    return defaultValue;
+  }
+  
+  private String getString(TextProperty textProperty) {
+    return getString(textProperty, null);
+  }
+  
+  private String getString(TextProperty textProperty, String defaultValue) {
+    if (textProperty != null) {
+      return getString(textProperty.getValue(), defaultValue);
+    }
+    
+    return null;
+  }
+
+  private String getFirst(List<? extends TextProperty> textProperties) {
+    if (textProperties != null && !textProperties.isEmpty()) {
+      return getString(textProperties.get(0));
+    }
+    
+    return null;
+  }
+
+  private List<String> getAll(List<? extends TextProperty> textProperties) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "getAll" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)